README.md

```diff
@@ -144,6 +144,21 @@ http://dockerhost:5000
 When you install docker-osx-dev, it adds an entry to your `/etc/hosts` file so
 that `http://dockerhost` works as a URL for testing your Docker containers.
 
+# docker-machine support
+
+docker-machine support is experimental. You can use it as the way it is used for
+Boot2Docker, but provide `--machine-name <machine-name>` as argument for 
+`docker-osx-dev`. So as an example, run as:
+
+```
+> cd /foo/bar
+> docker-osx-dev --machine-name <machine-name>
+[INFO] Performing initial sync of paths: /foo/bar
+[INFO] Watching: /foo/bar
+```
+
+Note: currently this assumes the docker-machine was already created.
```

Comment:
Instead of this note, could you include the `docker-machine create` command in the example above, similar to how you did in [this comment](https://github.com/brikis98/docker-osx-dev/pull/77#issuecomment-128207268)?